### ISSUE TITLE
Article rating is relevant only for the English Wikipedia

### ISSUE THREAD

**Issue by amire80** [WikiEducationFoundation/WikiEduDashboard#241]
Article rating according to WikiProjects is relevant only for the English Wikipedia. Is it clear to the dashboard's users? Applying these ratings to articles in other languages may be irrelevant. Wikipedias in some languages have a different scale, and most languages don't have WikiProjects and ratings at all.

**Comment by ragesoss** [comment]
This is true. The likely path forward initially will be to disable these ratings for other languages. (New scrapers could be implemented for individual languages that have similar systems.)

However, there's also the option of using ORES scores for other languages, to provide a simple estimate of how developed an article is: https://meta.wikimedia.org/wiki/Objective_Revision_Evaluation_Service

**Comment by ragesoss** [comment]
We've now switched to using the PageAssessments extension for this, which may open it up to other languages once the extension is enabled on them. Implementing fallback support for Wikidata badges may be a way to extend support further. See https://phabricator.wikimedia.org/T163556

**Comment by ragesoss** [comment]
The relevant places to start for adding support for Wikidata badges are:

https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/lib/importers/rating_importer.rb

and

https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/lib/wiki_api.rb

We should be able to configure which wikis have badges enabled and which ones have PageAssessments enabled.
